web-app/js/portal/cart/DownloadPanel.js

```diff
@@ -121,7 +121,34 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
 
         this._loadMenuItemsFromHandlers(processedValues, collection);
 
-        return this._applyTemplate(tpl, processedValues);
+        if (this._spatialSubsetIntersect(collection)) {
+            return this._applyTemplate(tpl, processedValues);
+        } else {
+            return this._emptyMsg();
+        }
+    },
+
+    _spatialSubsetIntersect: function(collection) {
```

Comment:
@jonescc just pointed out that if someone uses the service directly eg. marvl, then they should also get a sensible error message, which they won't if they bypass the portal.